client/src/pages/Feed.js

```diff
@@ -73,21 +74,6 @@ import GTM from "../constants/gtm-tags";
 import TagManager from "react-gtm-module";
 import WithSummitBanner from "components/WithSummitBanner";
 
-export const isAuthorOrg = (organisations, author) => {
```

Comment:
I move these two methods into utils instead of leaving it in "Feed Page" since this has nothing to deal with Feed, and just a util method. I also move other pages that require these methods to be importing from utils instead.